todo/todo/src/components/TodoItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+export const TodoItem = (props) => {
+
+    const {todoItem, setTodoItem, filter} = props;
+
+    const onClickDelete = (targetItem) => {
+        const newTodo = [...todoItem];
+        let targetIndex = newTodo.indexOf(targetItem);
+        newTodo.splice(targetIndex, 1);
+        setTodoItem(newTodo);
+    }
+    
+    const onClickStatus = (targetItem) => {
+        const newTodo = [...todoItem];
+        const item = newTodo.find((item => item.id === targetItem.id));
+        if(item.status){
+            item.status = false;
+        }else{
+            item.status = true;
+        }
+        let targetIndex = newTodo.indexOf(targetItem);
+        newTodo[targetIndex] = item;
+        setTodoItem(newTodo);
+    };
+
+    let displayItems = todoItem.filter( item => {
+        if (filter === 'all') return true;
+        if (filter === 'working') return !item.status;
+        if (filter === 'complete') return item.status;
+        return null;
+    });
+
+    return(
+        displayItems.map((Todo,index) => {
```

Comment:
mapの第一引数のTodoが先頭が大文字になっており不自然です。
また、mapのような配列ループの第一引数には一つずつ配列の値が入ることになりますので、元の配列の単数形を命名するのが一般的です。

もう一点、細かいですが、`,`の後ろにはスペースを空けましょう。
```suggestion
        displayItems.map((Todo, index) => {
```